packages/kuma-gui/src/app/x/components/x-empty-state/XEmptyState.vue

```diff
@@ -18,59 +18,67 @@
         ]"
         :key="title"
       >
-        <KEmptyState
-          class="x-empty-state"
+        <EntityEmptyState
+          :title="title"
+          appearance="secondary"
           data-testid="empty-block"
         >
-          <template
-            #icon
-          >
-            <slot name="icon" />
+          <template #image>
+            <div class="empty-state-icon">
+              <slot
+                v-if="slots.icon"
+                name="icon"
+              />
+              <component
+                :is="iconMapping[type]"
+                v-else-if="type && iconMapping[type]"
+                :color="KUI_COLOR_TEXT_DECORATIVE_AQUA"
+                :size="KUI_ICON_SIZE_50"
+              />
+              <AnalyticsIcon v-else />
+            </div>
           </template>
-          <template
-            #title
-          >
-            <slot
-              name="title"
-            >
-              <template
-                v-if="title.length > 0"
-              >
-                <header>
-                  <h2>
-                    <XI18n
-                      :path="`${prefix}x-empty-state.title`"
-                      default-path="components.x-empty-state.title"
-                    />
-                  </h2>
-                </header>
-              </template>
-            </slot>
+          <template #title>
+            <header v-if="title.length">
```

Comment:
Does this `v-if` need to go on the template like before? 

Looks like if we could get an empty div rendered if we leave it like this if I understood correctly?

https://github.com/Kong/public-ui-components/blob/b4f6446e145b962bdb6505a9790a8c4a4304677a/packages/entities/entities-shared/src/components/entity-empty-state/EntityEmptyState.vue#L11-L14

Reply:
updated